setup.cfg

```diff
@@ -4,7 +4,7 @@ description = Template tool for putting up the scaffold of a Python project
 author = Florian Wilhelm
 author-email = Florian.Wilhelm@gmail.com
 license = MIT
-url = https://github.com/blue-yonder/pyscaffold/
+url = https://github.com/pyscaffold/pyscaffold/
```

Comment:
I see that the project was moved to its own organization. Neat! :+1:

Reply:
Yeah, this is really cool. Blue Yonder agreed to move it over which gives us a bit more freedom in adding new contributors.